python/python/tests/test_transaction.py

```diff
@@ -626,8 +626,6 @@ async def transaction_test(
     args.append(b"one")
     transaction.srandmember_count(key7, 1)
     args.append([b"one"])
-    transaction.wait(1, 1000)
```

Comment:
why?

Reply:
I wrote in the description but i'll elaborate about the reason - "wait" has a flakey response in its nature as it requires the replicas to respond to the server in the specified timeout duration. If the replicas doesn't get to answer in time - which can happen in an integration test environment, we wouldn't get the expected result. The test already expect not to get a the replica to respond in time (expected value is 0), but we see in some tests that the return value is 1 meaning that the replica was able to response in the specified time. So indeed we see some flakeyness where the return value is 1 instead of 0. This command shouldn't be tested inside of the long transaction test as it makes the whole test flakey.